.mailmap

```diff
@@ -548,6 +548,7 @@ Elias Basler <e.e.basler@protonmail.com>
 Elisha Hollander <just4now666666@gmail.com> donno2048 <just4now666666@gmail.com>
 Elliot Marshall <Marshall2389@gmail.com> <marshall2389@gmail.com>
 Elrond der Elbenfuerst <elrond+sympy.org@samba-tng.org>
+Emile <emile.fourcin1@gmail.com>
```

Comment:
This should usually look something like:
```
Emile Fourcini <emile.fourcin1@gmail.com> Emile <emile.fourcin1@gmail.com>
```
The first part is the name that should go in the authors file and the second part is the name and email that is recorded in the commits.